infrastructure/template.yaml

```diff
@@ -103,6 +103,20 @@ Globals:
           - SecretArn: !FindInMap [Dynatrace, SecretArn, !Ref Environment]
 
 Resources:
+  SsmParametersFunction:
+    Type: AWS::Serverless::Function
+    Metadata:
+      BuildMethod: esbuild
+      BuildProperties:
+        Sourcemap: true
+    Properties:
+      Handler: lambdas/ssm-parameters/src/ssm-parameters-handler.lambdaHandler
+      CodeSigningConfigArn:
+        !If [EnforceCodeSigning, !Ref CodeSigningConfigArn, !Ref AWS::NoValue]
+      Policies:
+        - SSMParameterReadPolicy:
+            ParameterName: "*"
```

Comment:
@jkunle The reason we added a wildcard is so the lambda can read and cache any SSM parameter. Since this is only a READ permission I think this is OK however if we want to be strict then it might be better to do this closer to when we're dev finished as we might still be adding/removing SSM parameters.